__docs__/wonder-blocks-tabs/tabs.argtypes.ts

```diff
@@ -0,0 +1,32 @@
+import type {ArgTypes} from "@storybook/react";
+
+export default {
+    tabs: {
+        table: {
+            type: {
+                summary: "Array<TabItem>",
+                detail: "type TabItem = {|\n\tid: string,\n\tlabel: React.ReactNode,\n\tpanel: React.ReactNode\n|}",
+            },
+        },
+    },
+    "aria-label": {
+        description:
+            "If there is no visible label for the tabs, set aria-label to a label describing the tabs. Note: Either aria-label or aria-labelledby should be provided.",
+        table: {
+            type: {
+                summary: "string",
+            },
+            category: "Accessibility",
+        },
+    },
+    "aria-labelledby": {
+        description:
+            "If the tabs have a visible label, set aria-labelledby to a value that refers to the labelling element. Note: Either aria-label or aria-labelledby should be provided.",
+        table: {
+            type: {
+                summary: "string",
+            },
+            category: "Accessibility",
+        },
```

Comment:
Documenting aria-label and aria-labelledby props manually since Storybook isn't able to autogenerate prop docs from the type we use to ensure one of these is set (`AriaLabelOrAriaLabelledby` type in `tabs.tsx`)